tests/unit/utils/test_debug.py

```diff
@@ -267,6 +267,7 @@ def test_get_all_objects(self, stubs, monkeypatch):
 
             global object registry - 0 objects:
         """).rstrip('\n')
+        # pylint: enable=unused-variable
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.